src/locales/en/pages.yaml

```diff
@@ -732,60 +732,50 @@ addendum-50:
     For Krashna's 50th birthday, a lustrumbook was created (Dutch only). The addendum for this book can be found [here](/lustrum-50-addendum.pdf).
 
 owee:
-  title: Krashna at the OWee 2022
-  poster: /images/owee2022EN.png
+  title: Krashna at the OWee 2023
+  poster: /images/owee2023.jpeg
   text: |
-    ## Sunday August 21st
-    ##### Schie parade (17:00-19:00)
-      On Sunday afternoon Krashna will be participating in the Schieparade. This is part of the start of Delft introduction week where we will try to acquire your interest using our instruments, voice and music.
-    ## Monday August 22nd
-    ##### Information market (13:00-17:45)
-      Obviously we’ll be attending the traditional information market, where you can ask all your questions. So find our stand, come and have a chat!
-    ##### Krashna canon evening (20:00-22:00)
-      On Monday evening there is a night of canon singing on the program, where we shall sing several canon songs.. Are you interested in singing or want to discover whether you have any affinity with it? Join us at Cafe de Tango (in the city centre of Delft) where you can meet enthusiastic Krahsnese and sing while enjoying a drink (or two).
-    ## Tuesday August 23rd
-    ##### Yoga morning (9:30-10:30)
-      Are you interested in finding some peace and quiet after a few exhausting days of OWee? Krashna organizes a yoga morning from 9:30-10:30 where you can enjoy some classical music while relaxing your body. The location for this activity will be published via our social media.
-    ## Wednesday August 24th
-    ##### Activity Market and Krashna Maestro (11:00-15:00)
-      During the activity market at X you have another chance to come by our stand and ask for information about our association. Furthermore, several ensembles will perform on the stage, and you can try your ‘’conduct skills’’ during a one and only Krashna Maestro!
-    ##### Fiddle inn evening (21:00-23:00)
-    On Wednesday, the second-last night of the OWee, the Oweecie organizes another activity in Cafe de Tango. This time the theme is folk music. Do you have an instrument at your disposal or do you feel like singing? Come by and feel free to participate! We will make sure the needed sheet music is around. Of course, you can also have a drink and a nice chat!
+    ## Sunday August 13
+    ##### Schieparade
+    On Sunday afternoon Krashna will be participating in the Schieparade. This is part of the start of Delft introduction week where we will try to acquire your interest using our instruments, voice and music.
+    ## Monday August 14
+    ##### Information market
+    Obviously we’ll be attending the traditional information market, where you can ask all your questions. So come to our stand and decide wether Krashna Musika is right for you!
+    ## Tuesday August 15
+    ##### Twister tournament
```

Comment:
Add times in text too, not only in poster. This goes for all activities for which we know the time, and all languages.